docs/examples/automatically-blur-nsfw-images.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: Blur a NSFW image
+nav_order: 13
+parent: Examples
+---
+
+# Introduction
+We're going to be creating a workflow that takes an image via a URL and checks if it is safe for work then outputs the response either blurred or unblurred depending on the contents.
+
+We will need 'imagerecognition' to describe whether the image is safe for work, 'blur' container to blur the image and a simple requester container to get the contents of the url.
+
+```yaml
+id: check-image
+functions:
+- id: check
+  image: vorteil/imagerecognition:v2
+- id: blur
+  image: vorteil/blur:v1
+- id: request
+  image: vorteil/request:v5
+description: "Evaluates an image using Google Vision API"
+states:
+  # continued in next code block
+```
+
+## Google Vision
+This part fetches the image from the URL and uses the Google Vision AI to determine whether it is safe for work.
```

Comment:
You should make it clear that the definition below is a state definition. As an example you could start this sentence with. `First we need to define a state that fetches the image...`